src/bogart/AS_BAT_MarkRepeatReads.C

```diff
@@ -389,136 +364,56 @@ findThickestPrevRead(Unitig *tig, uint32 fi, int32 rMin, int32 rMax) {
     }
   }
 
-  //  Decide if the repeat region is to the left or right of this read.
-  //  Then ignore this thickest read if it doesn't fall in the repeat correctly.
-  //
-  //  A prev that ends inside the repeat, and so could be confused:
-  //
-  //              ----[rrrrrrrrrrrrrrrrrrrrrrrrrrrrr]------------------
-  //                                   ----------------------rdA
-  //                    rdB------------------
-  //
-  //  A prev that ends outside the repeat, can't be confused.
-  //
-  //              ----[rrrrrrrrrrrrrrrrrrrrrrrrrrrrr]------------------
-  //                                   ----------------------rdA
-  //                    rdB----------------------------
-  //
+  //  If this thickest overlap ends outside the repeat region, the overlap
+  //  cannot be confused.
 
-  assert(rdAlo <= rMax);
-  assert(rMin <= rdAlo);
+  ufNode *rdB   = nullptr;
+  int32   rdBlo = 0;
+  int32   rdBhi = 0;
 
-  int32    rdBlo = tig->ufpath[bestIdx].position.min();
-  int32    rdBhi = tig->ufpath[bestIdx].position.max();
+  if (bestIdx != UINT32_MAX) {
+    assert(rdAlo <= rMax);
+    assert(rMin <= rdAlo);
 
-  //  If the high coordinate isn't inside the repeat, we don't care;
-  //  rdB will be anchored to unique sequence.
-  //
-  //  For finding the previous read:
-  //
-  //        --------[repeat]]]--------
-  //                   -------------         READ WE'RE SEARCHING FROM
-  //                 ---------u              DON'T CARE, anchored in unique (u)
-  //          -----                          DON'T CARE, shouldn't happen
-  //
-  //  For the next read:
-  //
-  //        ------[[[repeat]----------
-  //       -------------                     READ WE'RE SEARCHING FROM
-  //             u---------                  DON'T CARE, anchored in unique (u)
-  //                          ------         DON'T CARE, shouldn't happen
-  //
-  //  The repeat is extended a bit so that the read will have some useful bit
-  //  of unique anchoring.
+    rdB   = &tig->ufpath[bestIdx];
+    rdBlo =  tig->ufpath[bestIdx].position.min();
```

Comment:
why isn't it rdB->position.min() and rdB->position.max() ?

Reply:
Why should it be?  I prefer things that line up nicely.  I could have just used rdB->position.max() throughout instead of assigning to a local variable.  If something non-obvious was going on, there would have been a comment about it.

Reply:
Well, first of all to make it more in line with how you defined the same values for A ;)

Reply:
Also, sure, you could have used rdB->position.max() , but I don't think you would use tig->ufpath[bestIdx]->position.max() already having rdB, right?

Reply:
Or how I set rdB just above.  Zero points for consistency here.